src/pages/home/ReportScreen.js

```diff
@@ -187,7 +187,8 @@ function ReportScreen({
 
     // There are no reportActions at all to display and we are still in the process of loading the next set of actions.
     const isLoadingInitialReportActions = _.isEmpty(reportActions) && reportMetadata.isLoadingInitialReportActions;
-    const isOptimisticDelete = lodashGet(report, 'statusNum') === CONST.REPORT.STATUS.CLOSED;
+
```

Comment:
NAB: I don't think adding a blank line here is necessary but not a huge deal

Reply:
Updated